### PR TITLE
Corrected up item_db

### PR DESCRIPTION
**Pull request by kaninhot004**
* **Addressed Issue(s)**: -

* **Server Mode**: Renewal

* **Description of Pull Request**: 

> 1132 - Adjusted coma target class to Class_All (https://www.divine-pride.net/database/item/1132)
> 1141 - Adjusted chance and remove MAGIC, MISC flags (https://www.divine-pride.net/database/item/1141)
> 1161 - Removed elemental and unbreakable weapon, Added int, luk bonues (https://www.divine-pride.net/database/item/1161)
> 1165 - Adjusted defense decrease rates (https://www.divine-pride.net/database/item/1165)
> 1166 - Adjusted damage bonus (https://www.divine-pride.net/database/item/1166)
> 1173 - Added chance to curse self when attack (https://www.divine-pride.net/database/item/1173)
> 1181 - Adjusted autobonus chance, Added sp cost reduction bonus, Remove vcast bonus (https://www.divine-pride.net/database/item/1181)
> 1187 - Adjusted damage bonus rates (https://www.divine-pride.net/database/item/1187)
> 1191 - Only Rune Knight equipable (https://www.divine-pride.net/database/item/1191)
> 1194 - Added damage taken from Demi-Human and ignore def to Demi-Human (https://www.divine-pride.net/database/item/1194)
> 1244 - Removed dex bonus, Added damage to demon bonus (https://www.divine-pride.net/database/item/1244)
> 1267 - Adjusted aspd rates (https://www.divine-pride.net/database/item/1267)
> 1274 - Removed elemental, Critical, UnbreakableWeapon bonus (https://www.divine-pride.net/database/item/1274)
> 1282 - Added critical damage rate bonus, Removed ignore def, Adjusted autobonus duration (https://www.divine-pride.net/database/item/1282)
> 1284 - Adjusted Sonic Blow chance (https://www.divine-pride.net/database/item/1284)
> 1298 - Implement item scripts and item_combo (https://www.divine-pride.net/database/item/1298)
> 1299 - Adjusted bleeding chance (https://www.divine-pride.net/database/item/1299)
>

### CODE REVIEW

**Comment by kaninhot004** [comment]
Updated

> 1306 - Added unbreakable bonus (https://www.divine-pride.net/database/item/1306)
> 1307 - Added unbreakable bonus (https://www.divine-pride.net/database/item/1307)
> 1314 - Added skill bonus (https://www.divine-pride.net/database/item/1314)
> 1315 - Added damage race bonus, Added skill bonus (https://www.divine-pride.net/database/item/1315)
> 1318 - Removed unbreakble bonus, Added skill bonus (https://www.divine-pride.net/database/item/1318)
> 1319 - Adjusted freeze rates (https://www.divine-pride.net/database/item/1319)
> 1321 - Removed unbreakble bonus, Added skill bonus (https://www.divine-pride.net/database/item/1321)
> 1322 - Adjusted available jobs (https://www.divine-pride.net/database/item/1322)
> 1336 - Implemented this item and combo (https://www.divine-pride.net/database/item/1336)
> 1378 - Change skill blow to skill damage bonus (https://www.divine-pride.net/database/item/1378)
> 1383 - Adjusted available jobs, Removed elemental bonus (https://www.divine-pride.net/database/item/1383)
> 1384 - Added autospell bonus (https://www.divine-pride.net/database/item/1384)
> 1385 - Adjusted available jobs (https://www.divine-pride.net/database/item/1385)
> 1386 - Move 2 bonus out of condition (https://www.divine-pride.net/database/item/1386)
> 1387 - Adjusted available jobs (https://www.divine-pride.net/database/item/1387)
> 1392 - Adjusted available jobs, Adjusted bonus rates (https://www.divine-pride.net/database/item/1392)
> 1393 - Adjusted available jobs, Adjusted effect to freeze (https://www.divine-pride.net/database/item/1393)
> 1394 - Added unbreakable bonus (https://www.divine-pride.net/database/item/1394)

**Comment by kaninhot004** [comment]
No more changes, Thanks

File: db/re/item_db.txt [hunk 1]
```diff
@@ -659,39 +659,41 @@
...
 1326,Illusion_War_Axe,Illusion War Axe,5,0,,4200,180,,1,2,0x00040420,63,2,2,4,120,1,6,{ bonus bStr,2; bonus bDex,2; bonus bLuk,2; .@r = getrefine(); if (.@r >= 9) { .@val = 40; } else if (.@r >= 7) { .@val = 20; } bonus2 bSkillAtk,"NC_AXETORNADO",(100+.@val); bonus2 bSkillAtk,"NC_AXEBOOMERANG",(100+.@val); bonus2 bSkillAtk,"GN_CART_TORNADO",(75+.@val); },{},{}
 //===================================================================
 // 2-Handed Axes
 //===================================================================
 1333,Golden_Wrench,Golden Wrench,5,,,5500,220,,1,2,0x00000400,56,2,2,4,170,1,6,{ .@r = getrefine(); bonus bUnbreakableWeapon; bonus2 bAddClass,Class_all,5; bonus bBaseAtk,4*.@r; if (.@r>=11) .@val = 35; else if (.@r>=9) .@val = 20; bonus2 bSkillAtk,"NC_AXEBOOMERANG",.@val; bonus2 bSkillAtk,"NC_POWERSWING",.@val; },{},{}
+1336,Guardian_Knight_Axe,Guardian Knight Axe,,0,,1500,210,,1,2,0xFFFFFBFF,56,2,2,4,170,1,6,{ bonus bUnbreakableWeapon; bonus bAtkRate,5; .@r = getrefine(); bonus bLongAtkRate,.@r; if(.@r >= 9){ bonus2 bSkillAtk,"NC_POWERSWING",20; } if(.@r >= 11){ bonus2 bAddRace,RC_Undead,20; bonus2 bAddRace,RC_Angel,20; } },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt:696]:
```suggestion
1336,Guardian_Knight_Axe,Guardian Knight Axe,5,0,,1500,210,,1,2,0xFFFFFBFF,56,2,2,4,170,1,6,{ bonus bUnbreakableWeapon; bonus2 bAddClass,Class_All,5; .@r = getrefine(); bonus bLongAtkRate,.@r; if(.@r >= 9){ bonus2 bSkillAtk,"NC_POWERSWING",20; } if(.@r >= 11){ bonus2 bAddRace,RC_Undead,20; bonus2 bAddRace,RC_Angel,20; } },{},{}
```

File: db/re/item_db.txt [hunk 2]
```diff
@@ -719,28 +721,28 @@
...
-1385,Bradium_Stonehammer,Bradium Stone Hammer,5,20,,2700,210,,1,0,0x000444A2,18,2,34,4,75,1,7,{ bonus3 bAddEffOnSkill,"BS_HAMMERFALL",Eff_Stun,500+(200*getrefine()); },{},{}
-1386,Doom_Slayer_I,Doom Slayer,5,0,,0,20,,1,0,0x000444A2,63,2,34,4,0,0,7,{ if(readparam(bStr)>=95){ bonus bBaseAtk,400; bonus2 bAddEff,Eff_Stun,3000; bonus bAspdRate,-25; bonus bUseSPrate,100; bonus bBreakArmorRate,500; } },{},{}
-1387,Giant_Axe,Giant Axe,5,20,,4000,330,,1,1,0x000444A2,18,2,34,3,50,1,7,{ bonus2 bSkillAtk,"WS_CARTTERMINATION",15; if(readparam(bStr)>=95) { bonus bHit,10; bonus bAspdRate,3; } },{},{}
+1383,Holy_Celestial_Axe,Celestial Axe,5,20,,1500,200,,1,0,0x00040420,63,2,34,4,60,1,7,{ bonus bVit,10; bonus2 bAddRace,RC_Undead,10; bonus3 bAutoSpell,"AL_BLESSING",5,30; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt:729]:
```suggestion
1383,Holy_Celestial_Axe,Celestial Axe,5,20,,1500,200,,1,0,0x00040420,63,2,34,4,60,1,7,{ bonus2 bAddRace,RC_Undead,10; bonus3 bAutoSpell,"AL_BLESSING",5,5; },{},{}
```
According to our db

**Comment by kaninhot004** [comment]
Thanks @Atemo !

Next times I will follow kRO description.

File: db/re/item_db.txt [hunk 3]
```diff
@@ -659,39 +659,41 @@
...
 1326,Illusion_War_Axe,Illusion War Axe,5,0,,4200,180,,1,2,0x00040420,63,2,2,4,120,1,6,{ bonus bStr,2; bonus bDex,2; bonus bLuk,2; .@r = getrefine(); if (.@r >= 9) { .@val = 40; } else if (.@r >= 7) { .@val = 20; } bonus2 bSkillAtk,"NC_AXETORNADO",(100+.@val); bonus2 bSkillAtk,"NC_AXEBOOMERANG",(100+.@val); bonus2 bSkillAtk,"GN_CART_TORNADO",(75+.@val); },{},{}
 //===================================================================
 // 2-Handed Axes
 //===================================================================
 1333,Golden_Wrench,Golden Wrench,5,,,5500,220,,1,2,0x00000400,56,2,2,4,170,1,6,{ .@r = getrefine(); bonus bUnbreakableWeapon; bonus2 bAddClass,Class_all,5; bonus bBaseAtk,4*.@r; if (.@r>=11) .@val = 35; else if (.@r>=9) .@val = 20; bonus2 bSkillAtk,"NC_AXEBOOMERANG",.@val; bonus2 bSkillAtk,"NC_POWERSWING",.@val; },{},{}
+1336,Guardian_Knight_Axe,Guardian Knight Axe,,0,,1500,210,,1,2,0xFFFFFBFF,56,2,2,4,170,1,6,{ bonus bUnbreakableWeapon; bonus2 bAddClass,Class_All,5; .@r = getrefine(); bonus bLongAtkRate,.@r; if(.@r >= 9){ bonus2 bSkillAtk,"NC_POWERSWING",20; } if(.@r >= 11){ bonus2 bAddRace,RC_Undead,20; bonus2 bAddRace,RC_Angel,20; } },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt:696]: The item type is missing
```
// ID,AegisName,Name,Type,
```
```suggestion
1336,Guardian_Knight_Axe,Guardian Knight Axe,5,0,,1500,210,,1,2,0xFFFFFBFF,56,2,2,4,170,1,6,{ bonus bUnbreakableWeapon; bonus2 bAddClass,Class_All,5; .@r = getrefine(); bonus bLongAtkRate,.@r; if(.@r >= 9){ bonus2 bSkillAtk,"NC_POWERSWING",20; } if(.@r >= 11){ bonus2 bAddRace,RC_Undead,20; bonus2 bAddRace,RC_Angel,20; } },{},{}
```

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**